content-src/components/StartupOverlay/StartupOverlay.jsx

```diff
@@ -12,11 +12,24 @@ export class _StartupOverlay extends React.PureComponent {
     this.initScene = this.initScene.bind(this);
     this.removeOverlay = this.removeOverlay.bind(this);
     this.onInputInvalid = this.onInputInvalid.bind(this);
+    this.setFxaMetrics = this.setFxaMetrics.bind(this);
 
     this.state = {
       emailInput: "",
-      overlayRemoved: false
+      overlayRemoved: false,
+      flowId: "",
+      flowBeginTime: ""
     };
+  }
+
+  componentWillUpdate() {
+    if (this.props.fxa_endpoint && !this.didFetch) {
```

Comment:
You need to define this.didFetch in the constructor with value of false

Reply:
sure, I figured undefined worked well enough